### PR TITLE
feat: add /health endpoint for monitoring purposes

### PR DESCRIPTION
**Pull request by elmurci**
Adding a health endpoint for monitoring purposes.

- Added `Express` as a dependency
- Created a new file defining the `/health` route
- Updated README

### CODE REVIEW

File: healthcheck.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+router.get('/', async (_req, res, _next) => {
+	const healthcheck = {
+		uptime: process.uptime(),
+		message: 'OK',
+		timestamp: Date.now()
+	};
+	try {
+		res.send(healthcheck);
+	} catch (e) {
```
Comment (intelliot) [inline healthcheck.js:12]: When does res.send throw?

File: healthcheck.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		healthcheck.message = e;
```
Comment (intelliot) [inline healthcheck.js:13]: It doesn't look like this is used

Comment (elmurci) [reply healthcheck.js:13]: you are right, I have pushed a change for this.

File: healthcheck.js (hunk 1 above)
Comment (elmurci) [reply healthcheck.js:12]: https://github.com/expressjs/express/blob/5596222f6a6e0eea74f6fb38915b071d98122a2f/lib/response.js#L107

We can catch a potential exception here maybe?

File: README.md [hunk 3]
```diff
@@ -14,3 +14,7 @@ Or to monitor the [XRP Ledger Test Net](https://ripple.com/build/xrp-test-net/)
 ````
 ALTNET=true WEBHOOK_URI=<your-slack-webhook-uri> npm start
 ````
+
+## Health Endpoint
+
+Available on `/health`.
```
[the remark this answers is not in the document]
Comment (elmurci) [reply README.md]: fixed

File: healthcheck.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router({});
+
+router.get('/', async (_req, res, _next) => {
```
Comment (intelliot) [inline healthcheck.js:4]: nit: if `_next` is unused, it can be removed

```suggestion
router.get('/', async (_req, res) => {
```

**Pull request merged**